autosportlabs/racecapture/datastore/datastore.py

```
@@ -603,7 +752,12 @@ def get_channel_smoothing(self, channel):
             return res[0]
 
     @timing
-    def import_datalog(self, path, name, notes='', progress_listener=None):
+    def import_datalog(self, path, name, notes='', progress_cb=None):
+        
+        warnings = []
+        if not self._isopen:
+            raise Exception("Datastore is not open")
```

Might be better to create a new Exception class that's specific to this scenario so other code can catch it if needed instead of having to catch all Exceptions.